TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class ColorPair
+    {
+        public Color majorColor;
+        public Color minorColor;
+        
+        public override string ToString()
+        {
+            return string.Format("MajorColor:{0}, MinorColor:{1}", majorColor.Name, minorColor.Name);
+        }
+        
+        public static int GetPairNumberFromColor(ColorPair pair)
+        {
+            // Find the major color in the array and get the index
+            int majorIndex = -1;
+            for (int i = 0; i < Program.colorMapMajor.Length; i++)
+            {
+                if (Program.colorMapMajor[i] == pair.majorColor)
+                {
+                    majorIndex = i;
+                    break;
+                }
+            }
+
+            // Find the minor color in the array and get the index
+            int minorIndex = -1;
+            for (int i = 0; i < Program.colorMapMinor.Length; i++)
```

Comment:
This `for` loop is almost a duplicate of the previous one. Resolve the duplication